kumascript/macros/ListSubpagesForSidebar.ejs

```diff
@@ -34,111 +34,93 @@ const overview = mdn.localString({
     'ko': '개요',
     'ru': 'Обзор',
     'zh-CN': '概述',
+    'zh-TW': '概述'
```

Comment:
poke @irvin for a review on zh-TW translation

Reply:
ping @irvin